### PR TITLE
Added tags and changelog file.

### PR DESCRIPTION
**Pull request by QBY-EstebanValverde**
# Description

Added tags variable for tagging resources.

# Change overview (tick true):

- [ ] This introduces backward incompatible changes
- [X] This adds a new backward compatible Feature
- [ ] This fixes a Bug

# Version information: 

- Previous Version: `1.0.5`

- Next Version based on [Semantic Versioning](https://semver.org/#summary) (see above): `1.1.0`

# How Has This Been Tested?

- [X] Apply of all examples was successfull
- [ ] Test B

# Checklist:

- [X] I have run tests and documented them above
- [X] I have performed a self-review of my own code
- [X] I have updated the documentation
- [X] I have updated the CHANGELOG

### CODE REVIEW

File: examples/basic/main.tf [hunk 1]
```diff
@@ -30,4 +30,7 @@ module "azureBastion" {
   sku                         = "Basic"
   # Following variables are optional
   copy_paste_enabled          = true
+  tags                        = {
+    "env" = "TST"
+  }
```
Comment (ghost) [inline examples/basic/main.tf]:
```suggestion
```
Basic examples should be as short as possible. At least that's the idea.

File: examples/standard/main.tf [hunk 2]
```diff
@@ -35,4 +35,7 @@ module "azureBastion" {
   file_copy_enabled           = true
   tunneling_enabled           = false
   shareable_link_enabled      = false
+  tags                        = {
+    "env" = "PRD"
```
Comment (ghost) [inline examples/standard/main.tf]:
```suggestion
    "tagname" = "tagvalue"
```

Let's not use real env in examples :D

File: variables.tf [hunk 3]
```diff
@@ -72,4 +72,10 @@ variable "shareable_link_enabled" {
   description = "Option in PREVIEW!!! Allow users connect to a target resource using Azure Bastion without accessing the Azure portal, only available in the Standard tier"
   type        = bool
   default     = null
+}
+
+variable "tags" {
+  type        = map(string)
+  description = "If specified, will set tags for all resources deployed by this module where supported."
+  default     = {}
```
Comment (ghost) [inline variables.tf]:
```suggestion
  default     = null
```
It's usually better to use null, as tf treats that as if the property wasn't used in the first place. It doesn't really matter for tags though, as the default value for tags is `{}`

**Review by ghost — DISMISSED** [review]
LGTM, comments just nitpicking.

File: examples/standard/main.tf (hunk 2 above)
Comment (QBY-EstebanValverde) [reply examples/standard/main.tf]: Applied recommendation.

File: variables.tf (hunk 3 above)
Comment (QBY-EstebanValverde) [reply variables.tf]: Applied recommendation.

**Review by ghost — APPROVED** [review]

**Pull request merged**